cirq-core/cirq/ops/projector.py

```diff
@@ -0,0 +1,161 @@
+import itertools
+from typing import (
+    Any,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Optional,
+    Union,
+)
+
+import numpy as np
+from scipy.sparse import coo_matrix
+
+from cirq import value
+from cirq.ops import raw_types
+
+
+def check_qids_dimension(qids):
```

Comment:
Nit: underscore at the beginning. We don't necessarily want this being a module level function for `projector.py`

Reply:
Done.